lib/rdf.js

```diff
@@ -127,6 +127,28 @@ function createRDF(manifest) {
     });
   }
 
+  if (manifest.locales) {
+    for (var locale in manifest.locales){
```

Comment:
oops, sorry for that, it's just a typo, but its creates xpi and signs it correctly so I haven't noticed that